coral/src/domain/topic/topic-types.ts

```diff
@@ -14,6 +14,12 @@ type TopicApiResponse = ResolveIntersectionTypes<Paginated<Topic[]>>;
 type Topic = KlawApiModel<"TopicInfo">;
 type TopicNames = KlawApiResponse<"getTopicsOnly">;
 type TopicTeam = KlawApiModel<"TopicTeamResponse">;
+type TopicMessages = {
+  [key: string]: string | undefined;
+};
+type NoContent = {
```

Comment:
Was this possible response discussed with the backend team? Seems a little unexpected :o Also, looking at the `openapi` reponse type, I think they wanted to improve the ones that are plain objects like this

```json
              "application/json" : {
                "schema" : {
                  "type" : "object",
                  "additionalProperties" : {
                    "type" : "string"
                  }
                }
              }
```

Reply:
Yeah, imo it's a bit odd, i raised it couple times but it seems this is how it works for now. 

The type here is technically correct, but there are basically 2 different meanings to the response:
{"status": false}  = no messages or consuming failed
{0: "hello", 1: "world"} = 2 messages

IMO it could work as a list better, empty list meaning no content then some structured message along with possible metadata, like the partition, timestamp, offset etc: `[ {"offset": "", "timestamp": "", "content": "" } ]`

@aindriu-aiven raised that it would be beneficial to expose more of the metadata, like the partition. This would be only possible by changing the structure though (for example the above)

I think that is something we can look into later. Now it basically works like in the old ui so should be good.